dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -506,7 +510,29 @@ export default defineComponent({
               const receivedFormInputsModel = convertValuesFromDecimalsToPercentages(
                 companyAssociatedEuTaxonomyData as ObjectType
               );
+
+              const kpiSectionsModel = Object.keys(euTaxonomyKPIsModel.kpisFieldNameToFinancialServiceType)
+                .map( key => {
+                  const fieldName = euTaxonomyKPIsModel.kpisFieldNameToFinancialServiceType[key];
+                  const kpi = (receivedFormInputsModel.data as ObjectType).eligibilityKpis[fieldName];
+                  const eligibilityKpi = kpi ? {[fieldName]: kpi } : null;
+                  return { [key]: eligibilityKpi };
+                })
+                .map( item => {
+                  const key = Object.keys(item)[0];
+                  const kpi = receivedFormInputsModel.data[key];
+                  if(kpi){
+                    item[key] = {...item[key], ...receivedFormInputsModel.data[key]};
```

Comment:
You could use kpi here instead of receivedFormInputsModel.data[key]

Reply:
Hrm. yes.. I think that was my intention. Nice.